src/routes/transactions/helpers/timezone.helper.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Converts a given date to a specified timezone.
+ *
+ * @param {Date} date The date object to be converted.
+ * @param {string} timeZone The target timezone (e.g., "Europe/Berlin").
+ *
+ * @returns {Date} A new Date object representing the date converted to the specified timezone
+ * @throws {RangeError} Throws if an invalid timezone is sent
+ */
+export const convertToTimezone = (date: Date, timeZone: string): Date => {
+  const convertedDate = new Intl.DateTimeFormat('en-US', {
+    timeZone,
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  }).format(date);
+
+  const [month, day, year] = convertedDate.split('/').map(Number);
+  const zeroBasedMonth = month - 1; // JavaScript months are zero-indexed (0 for January, 11 for December), so we subtract 1
+
+  return new Date(Date.UTC(year, zeroBasedMonth, day));
+};
```

Comment:
This works for our usecase but just for future reference: this converts the date but not the time to the given timezone.